IceCubesApp/Resources/Localization/nb.lproj/Localizable.strings

```diff
@@ -291,6 +291,7 @@
 
 // MARK: Package: Status
 "status.action.translate" = "Oversett";
+"status.action.translate-from-%@" = "Translate from %@";
```

Comment:
norwegian?

Reply:
I have translated the new strings (including this) for Norwegian in PR #537